src/librustc_codegen_llvm/base.rs

```diff
@@ -42,7 +42,7 @@ use rustc::hir::CodegenFnAttrs;
 use crate::value::Value;
 
 pub fn write_compressed_metadata<'a, 'gcx>(
```

Comment:
`'a` isn't necessary anymore

Reply:
That's weird, why wouldn't it get an error?

Reply:
Oh, the codegen crates are missing `#![deny(internal)]` and `#![deny(unused_lifetimes)]`...